src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -668,6 +675,9 @@ impl CompleteStreamFragmentGraph {
 
                             (mview_id, edge)
                         }
+                        DdlType::Source | DdlType::Table(_) => {
```

Comment:
Is this branch reachable for schema change on a sink-into table?

Reply:
Can you elaborate?

Reply:
https://github.com/risingwavelabs/risingwave/issues/14417 Oh, it's a TODO now. Never mind.